FAQ.md

```diff
@@ -203,3 +203,71 @@ targets:
 ```
 
 Also, you can also check out the example configuration in the Mockito repository. 
+
+
+## How do I mock a `sealed` class?
```

Comment:
This section discusses how to mock a class that requires instances of sealed classes, rather that mocking sealed classes.

Reply:
Oops.  Heh, I even made a similar correction to a StackOverflow question about this.

I also updated the URLs referring to this section. I feel that the new heading is too long and is more brittle (and I don't think GitHub Markdown supports generating custom slugs?), so I've changed the URLs to be a bit less specific.